Makefile

```diff
@@ -1,35 +1,59 @@
+# useful paths
+SRC_DIR = src
+HDR = $(SRC_DIR)/cs50.h
+SRC = $(SRC_DIR)/cs50.c
+
+BUILD_DIR = build
+OBJ = $(BUILD_DIR)/cs50.o
+USR_DIR = $(BUILD_DIR)/usr
+INCLUDE_DIR = $(USR_DIR)/include
+LIB_DIR = $(USR_DIR)/lib
+MAN_DIR = $(USR_DIR)/share/man/man3
+BUILD_SRC = $(USR_DIR)/src
+
+# eprintf
+EPRINTF_EXE = $(BUILD_DIR)/eprintf
+
+# HackerRank
+HR_HDR = $(BUILD_DIR)/cs50.h
+HR_EXE = $(BUILD_DIR)/hackerrank
+
+TESTS_DIR = tests
+
+# deb package info
 DESCRIPTION = CS50 Library for C
 MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
 NAME = lib50-c
 OLD_NAME = library50-c
-VERSION = 7.0.2
+VERSION = 7.1.0
```

Comment:
Why the middle version bump? Have we added something new to the library itself? Or just the build process?